include/pokemon.h

```diff
@@ -176,18 +176,20 @@ typedef struct BoxPokemon {
 
 /**
  * @brief Party Pokemon data structure
+ *  This is used to store stats for a pokemon while it is in the players party.
+ *  Rather than recalculating stats after each battle, they're stored here.
  */
 typedef struct PartyPokemon {
-    u32 unk_00;
-    u8 level;  //!< The pokemons level
-    u8 unk_05;
-    u16 unk_06;
-    u16 unk_08;
-    u16 unk_0A;
-    u16 unk_0C;
-    u16 unk_0E;
-    u16 unk_10;
-    u16 unk_12;
+    u32 status;
+    u8 level;
+    u8 mail;
```

Comment:
Is this mail? Or an item?

Reply:
I noticed the data structure matched the previous games, but I believe it to be the ID of the mail the pokemon is holding